index.html

```diff
@@ -12,27 +12,37 @@
 
 <body>
     <main>
+        <header>
+            <nav>
+                <a href="#headline" id="gabriel">Gabriel</a>
+                <ul class="tab tab-nav">
+                    <li class="tab-link">
+                        <a href="#portfolio">Portfolio</a>
+                    </li>
+                    <li class="tab-link">
+                        <a href="#about">About</a>
+                    </li>
+                    <li class="tab-link">
+                        <a href="#contact">Contact</a>
+                    </li>
+                </ul>
+                <div class="hamburger ham" id="hamburger">
+                    <div class="bar-1"></div>
+                    <div class="bar-2"></div>
+                    <div class="bar-3"></div>
+                </div>
+            </nav>
+            <nav class="curtain curtain-anim" id="curtain">
+                <ul class="tab-items">
+                    <li id="nav1"><a href="#portfolio" class="mobile-links">Portfolio</a></li>
+                    <li id="nav2"><a href="#about" class="mobile-links">About</a></li>
+                    <li id="nav3"><a href="#contact" class="mobile-links">Contact</a></li>
```

Comment:
- [ ] Please note that when the user clicks (or taps) any of the mobile menu options, a correct part of the page should be displayed. currently, your links don't work so kindly try to fix that

![linkkkk](https://user-images.githubusercontent.com/89282221/190706597-6294418b-f551-4736-a28d-1576a6ddf951.gif)